lib/rules/at-rule-descriptor-no-unknown/__tests__/index.mjs

```diff
@@ -0,0 +1,106 @@
+import rule from '../index.mjs';
+const { messages, ruleName } = rule;
+
+testRule({
+	ruleName,
+	config: true,
+
+	accept: [
+		{
+			code: '@counter-style foo { system: cyclic; symbols: url(gold-medal.svg); suffix: " "; }',
+		},
+		{
+			code: '@property --foo { syntax: "<integer>"; inherits: false; initial-value: 0; }',
+		},
+		// TODO: The following test can be supported once mdn/data, is updated and the css-tree is updated.
+		// see: https://github.com/mdn/data/blob/6b4cae7c0aa96c199a982f2b6efeda7ce67eb515/css/at-rules.json
+		// {
+		// 	code: '@view-transition { navigation: auto; }',
+		// },
+		// {
+		// 	code: '@container (width > 400px) and (height > 400px) { a { width: 50%; } }',
+		// },
+		// {
+		// 	code: '@container not (width > 400px) { a { width: 50%; } }',
+		// },
+		// {
+		// 	code: '@container summary (min-width > 400px) { @container (min-width: 800px) { a { width: 50%; } } }',
+		// },
```

Comment:
```suggestion
		code: '@media (width > 400px) { a { color: red; } }',
```

This should pass as `color` is a property, not a descriptor.

[Test conventions](https://stylelint.io/developer-guide/rules#write-tests).